tweeter_user.py

```diff
@@ -0,0 +1,151 @@
+from datetime import datetime
+import re
+import csv
+from hashlib import sha256
+import os
+
+USER_DETAILS = "details.csv"
+
+class User:
+
+    """
+    @Class User - Generic class representing the user in the tweeter application
+    The class holds all the properties and methods required to fully manipulate a user inside the class.
+    """
+    def __init__(self, **kwargs):
```

Comment:
public user property name will allow external modification of internal object data
@George Koomson